src/components/VmDetails/BaseCard.js

```diff
@@ -146,6 +161,7 @@ BaseCard.propTypes = {
   editable: PropTypes.bool,
   disableSaveButton: PropTypes.bool,
   editTooltip: PropTypes.string,
+  disableTooltip: PropTypes.string,
```

Comment:
`disableTooltip` name is a bit confusing.  It is to disable the edit button's tooltip or is it a particular tooltip to display?